Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -766,88 +771,503 @@ def add_user_to_channel_command():
+        allow_team_mentions=argToBoolean(demisto.args().get('allow_team_mentions', 'false')),
+        allow_channel_mentions=argToBoolean(demisto.args().get('allow_channel_mentions', 'false')),
+    )
+    demisto.results(f'Team {display_name} was created successfully.')
+
+
+def create_team_from_group_request(
+        group_id: str,
+        display_name: str,
+        description: Optional[str] = None,
+        visibility: str = 'public',
+        allow_guests_create_channels: bool = False,
+        allow_guests_delete_channels: bool = False,
+        allow_members_create_private_channels: bool = False,
+        allow_members_create_channels: bool = False,
+        allow_members_delete_channels: bool = False,
+        allow_members_add_remove_apps: bool = False,
+        allow_members_add_remove_tabs: bool = False,
+        allow_members_add_remove_connectors: bool = False,
+        allow_user_edit_messages: bool = False,
+        allow_user_delete_messages: bool = False,
+        allow_owner_delete_messages: bool = False,
+        allow_team_mentions: bool = False,
+        allow_channel_mentions: bool = False,
+) -> None:
+    url = f'{GRAPH_BASE_URL}/v1.0/groups/{group_id}/team'
+    http_request(
+        'PUT',
+        url,
+        json_={
+            'displayName': display_name,
+            'description': description,
+            'visibility': visibility,
+            'guestSettings': {
+                'allowCreateUpdateChannels': allow_guests_create_channels,
+                'allowDeleteChannels': allow_guests_delete_channels,
+            },
+            'memberSettings': {
+                'allowCreatePrivateChannels': allow_members_create_private_channels,
+                'allowCreateUpdateChannels': allow_members_create_channels,
+                'allowDeleteChannels': allow_members_delete_channels,
+                'allowAddRemoveApps': allow_members_add_remove_apps,
+                'allowCreateUpdateRemoveTabs': allow_members_add_remove_tabs,
+                'allowCreateUpdateRemoveConnectors': allow_members_add_remove_connectors,
+            },
+            'messagingSettings': {
+                'allowUserEditMessages': allow_user_edit_messages,
+                'allowUserDeleteMessages': allow_user_delete_messages,
+                'allowOwnerDeleteMessages': allow_owner_delete_messages,
+                'allowTeamMentions': allow_team_mentions,
+                'allowChannelMentions': allow_channel_mentions,
+            },
+        },
+    )
+
+
+def create_team_from_group():
+    group_id = demisto.args().get('group_id', '')
+    create_team_from_group_request(
+        group_id=group_id,
+        display_name=demisto.args().get('display_name'),
+        description=demisto.args().get('description'),
+        visibility=demisto.args().get('visibility', 'public'),
+        allow_guests_create_channels=argToBoolean(demisto.args().get('allow_guests_create_channels', 'false')),
```

Comment:
can we switch to the same format as in `update_team`?

Reply:
the difference between create-team commands and update command is that in the update command, i dont want to the bool args to have default value, as they might cause the user to update properties which were not meant to be updated - if that wasnt the case, i would implement the update command the same as the create commands
i can modify the create commands to be implemented in the same style as the update command is implemented, but i think the way it is more straight forward and simple
what do you think?

Reply:
it's fine as is